packages/material-ui/src/styles/createSpacing.d.ts

```diff
@@ -15,6 +15,6 @@ export interface Spacing {
   ): string;
 }
 
-export type SpacingOptions = number | ((factor: number) => string | number) | number[];
+export type SpacingOptions = number | ((factor: number) => string | number) | SpacingArgument[];
```

Comment:
We need to add a test for it in `createSpacing.test.js`, I have no idea if it's supported